README.md

```diff
@@ -159,6 +159,8 @@ RedisSessionHandler solves this problem with a "lock" entry for every session th
 [Session fixation](https://www.owasp.org/index.php/Session_fixation) is the ability to choose your own session ID as an HTTP client. When clients are allowed to choose their
 session IDs, a malicious attacker might be able to trick other clients into using an ID already known to him, then wait for them log in and hijack their session.
 
+**To ensure cookies are replaced, any custom cookie parameters should be registered with session_set_cookie_params() as browsers will only update cookies with matching parameters.**
```

Comment:
If it works however the parameters are specified, then it is not necessary. I believe it will now work as parameters will always come from session_get_cookie_params() regardless. Your screenshot shows that session_start respects session.* parameters but setcookie only does this when specified explicitly. I am wondering whether there is a function we should actually be using such as session_regenerate_id() as then we are not coupling the session handler to usage of cookies. It remains to check whether this sends a new cookie but I will try it now.

I am trying to see if we have covered all the comments in the issue you referenced. I think the plugin is quite stable for most standard use cases now but testing may be required if you want to do something unusual (like run code with no time limit!). There are some knowledgeable comments but no pull requests!